### DISCUSSION TITLE
Adding to video sources. (Extra audio tracks, subs, etc)

### DISCUSSION THREAD

**Discussion by TERMINATOR461997** [cdgriffith/FastFlix#623] [Q&A] [answered]
This may be a dumb question, but I cant seem to get Fastflix to add an audio track to a project. For example, I'm working on adding Japanese audio and English subs to a AV1 video, but Fastflix just spits out a warning about it not being a video file. Is there something I am missing here? 

Going through command line ffmpeg is very annoying to do, since there is no complete list of the commands and what they do that I have found. Most are the basics, and the documentation doesn't list the commands in a easy way to locate the command for what your trying to do.  (Feel free to correct me)

**Comment by cdgriffith** [comment, marked as answer]
Currently there is no way to merge multiple files into a single one with FastFlix, would be a great feature but a lot of work
